packages/backend-app-api/src/services/implementations/tokenManager/tokenManagerFactory.ts

```diff
@@ -31,6 +31,6 @@ export const tokenManagerFactory = createServiceFactory({
     const tokenManager = ServerTokenManager.fromConfig(config, {
       logger,
     });
-    return async () => tokenManager;
+    return tokenManager;
```

Comment:
Just noting that this is a change of behavior; it'll make multiple instances now but didn't before. Maybe that's safe here

Reply:
Hmm does this need moving to `createRootContext` then I wonder? Didn't we have a similar problem last week with duplicated `TokenManagers`?

Reply:
👍 , fixed + added a regression test